src/core/midi/MidiOss.cpp

```diff
@@ -39,8 +39,10 @@ MidiOss::MidiOss() :
 {
 	// only start thread, if opening of MIDI-device is successful,
 	// otherwise isRunning()==false indicates error
-	if( m_midiDev.open( QIODevice::ReadWrite ) ||
-					m_midiDev.open( QIODevice::ReadOnly ) )
+	if( m_midiDev.open( QIODevice::ReadWrite |
+		QIODevice::Unbuffered ) ) ||
```

Comment:
Oops, I missed a typo which breaks build.
```suggestion
		QIODevice::Unbuffered ) ||
```

Reply:
Fixed in 877a93198f0793fb6e236c3694c7a02fc2527e57.

Reply:
Thanks!